mobile/src/screens/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -32,7 +32,10 @@ const SignInForm: React.FC<Properties> = ({ onSubmit }) => {
 
   return (
     <React.Fragment>
-      <ScrollView style={styles.formContainer}>
+      <ScrollView
+        style={styles.formContainer}
+        showsVerticalScrollIndicator={false}
```

Comment:
why do we need this?
```suggestion
        showsVerticalScrollIndicator={false}
```

Reply:
@Liza-Veis it shows this scroll bar on sign in screen , and we dont have one on sign up screen. For UI it looks fetter without this scroll bar
![Снимок экрана 2023-09-07 122916](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/111873795/523a0e98-7565-4e37-bb1a-ffc05514ca1a)

Reply:
1. why don't we have it on sign up if we use same layouts? 
2. as for me, we should leave it but the padding should be inside scrollview, not outside, so that we have this scroll near the right side

Reply:
@Liza-Veis okay, i will remove these changes )

Reply:
@Liza-Veis maybe it would be better then to create ticket for styling this scroll bar in sign in and sign up ?

Reply:
yes